sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -367,6 +367,47 @@ def has_ontime_pane(self):
     return False
 
 
+class Always(TriggerFn):
+  """Repeatedly invoke the given trigger, never finishing."""
+  def __init__(self):
+    pass
+
+  def __repr__(self):
+    return 'Always'
+
+  def __eq__(self, other):
+    return type(self) == type(other)
+
+  def __hash__(self):
+    return 1
```

Comment:
More conventional is to return hash(type(self)), to avoid collisions, but this is OK.